std_msgs/Header header
float32 resolution_x
float32 resolution_y
sensor_msgs/Image map